### DISCUSSION TITLE
How can I create a "Jupyter Lab" shortcut like you get when installing anaconda?

### DISCUSSION THREAD

**Discussion by kienerj** [conda-forge/miniforge#467] [Q&A] [answered]
This is for Windows. As title says. When installing anaconda, I get a nice shortcut that launches jupyter. I tried to "emulate" it by creating it myself with mambaforge but it doesn't work (as a windows opens but closes again immediately, no error visible).

This is the content of the shortcut (the "Target field"):

`%USERPROFILE%\AppData\Local\mambaforge\python.exe %USERPROFILE%\AppData\Local\mambaforge\cwp.py %USERPROFILE%\AppData\Local\mambaforge %USERPROFILE%\AppData\Local\mambaforge\python.exe %USERPROFILE%\AppData\Local\mambaforge\Scripts\jupyter-lab-script.py "E:\notebooks"`

**Comment by jaimergp** [comment]
Shortcuts are created using `menuinst`. Read here:

* https://github.com/conda/menuinst/wiki/Menus-in-Conda-Recipes

Miniforge's shortcut is provided by the [`miniforge_console_shortcut`](https://github.com/conda-forge/miniforge_console_shortcut-feedstock/tree/main/recipe).

**Comment by kienerj** [comment, marked as answer]
Found the solution on stackoverflow (not all solution sin that treahd actually work for mambaforge)

    %windir%\System32\cmd.exe "/K" %USERPROFILE%\AppData\Local\mambaforge\Scripts\activate.bat base & jupyter lab && exit

of course one needs to adjust to the correct path and environment (I like to have base env run jupyter but notebooks using a different kernel (different env).
